.github/workflows/benchmark.yml

```
@@ -229,8 +234,8 @@ jobs:
         uses: CodSpeedHQ/action@3194d9a39c4d46684cb44bf7207fc56626aad8fd # v4.15.1
         timeout-minutes: 50
         with:
-          mode: simulation
-          run: cargo codspeed run
+          mode: ${{ matrix.mode || 'simulation' }}
+          run: cargo codspeed run ${{ matrix.bench }}
```

actionlint (1.7.12)

[error] property "bench" is not defined in object type {package: string}

(expression)